src/app/project/cac/become-a-member.component.html

```diff
@@ -93,17 +93,17 @@ <h5 class="modal-title">Member Sign-up</h5>
 
             <!-- I live near -->
             <div class="form-group row">
-                <label class="control-label"><input class="mr-1" name="liveNear" type="checkbox" [(ngModel)]="liveNear"/>I live near or use the project area</label>
+                <label class="control-label"><input class="mr-1" name="liveNear" type="checkbox" [(ngModel)]="liveNear"/>I live near or have particular knowledge of the project area</label>
                 <input [hidden]="!liveNear" class="form-control col-12" type="text" name="liveNearInput" id="liveNearInput" [(ngModel)]="liveNearInput" autofocus />
             </div>
             <!-- I am a member of -->
             <div class="form-group row">
-              <label class="control-label"><input class="mr-1" name="memberOf" type="checkbox" [(ngModel)]="memberOf"/>I am a member of a stakeholder organization</label>
+              <label class="control-label"><input class="mr-1" name="memberOf" type="checkbox" [(ngModel)]="memberOf"/>I am a member of a organization with an interest in the project (provide organization name below)</label>
```

Comment:
I am a member of a**_n_** organization with an interest in the project (provide organization name below)